### ISSUE TITLE
Error finding Cython dependency on pip install

### ISSUE THREAD

**Issue by FriedLabJHU** [idptools/metapredict#13]
I received the following error when trying to install via pip on python 3.9.0. 3.10.11, and 3.11.3. Cython is already installed.

```
Collecting metapredict
  Using cached metapredict-2.62.tar.gz (1.6 MB)
  Installing build dependencies ... done
  Getting requirements to build wheel ... error
  error: subprocess-exited-with-error

  × Getting requirements to build wheel did not run successfully.
  │ exit code: 1
  ╰─> [8 lines of output]
      ########################################
     
      Error: Please install cython first:
     
      pip install cython
     
      ########################################
     
      [end of output]

  note: This error originates from a subprocess, and is likely not a problem with pip.
error: subprocess-exited-with-error

× Getting requirements to build wheel did not run successfully.
│ exit code: 1
╰─> See above for output.

note: This error originates from a subprocess, and is likely not a problem with pip.
```

**Comment by ryanemenecker** [comment]
Sorry to hear you're having issues with metapredict! I just made a clean virtual environment and pip installed metapredict, so I wasn't able to reproduce your issue. There are a few things we can try though, I just need some more info on your end. Could you please post the output from running the following command?
`pip show cython`
Also, are you using a conda environment? If you are, could you try:
`conda create --name test_env python=3.9`
`conda activate test_env`
`pip install metapredict`

**Comment by ryanemenecker** [comment]
Also, if you want to email me for additional troubleshooting, feel free to at remenecker@wustl.edu

**Comment by FriedLabJHU** [comment]
I am not using conda or a venv, is it recommended to run metapredict in the venv every time?

Here is the output for """pip show cython"""

"""
>py -3.9 -m pip show cython                                                                
Name: Cython
Version: 3.0.0
Summary: The Cython compiler for writing C extensions in the Python language.
Home-page: https://cython.org/
Author: Robert Bradshaw, Stefan Behnel, Dag Seljebotn, Greg Ewing, et al.
Author-email: cython-devel@python.org
License: Apache-2.0
Location: c:\program files\python39\lib\site-packages
Requires:
Required-by:
"""

**Comment by ryanemenecker** [comment]
I tend to recommend using a virtual environment; although, it shouldn't be necessary. In your install, it says that it's using a cached version, so it might not be compiling the Cython code correctly. Could you try:
`pip uninstall metapredict`
Just to make sure you don't have metapredict installed. This won't do anything if you didn't manage to install it...
Then...
`git clone https://github.com/idptools/metapredict.git`
`cd metapredict`
`pip install -e .`
By cloning the repo and installing locally **and using the -e flag when pip installing**, it should make sure that the cython code compiles correctly.

**Comment by ryanemenecker** [comment]
Hi FriedLabJHU,
I just wanted to follow up to see if you managed to get metapredict to work. If you did, can you please tell me how you got it to work so I can add it to our troubleshooting?

**Comment by FriedLabJHU** [comment]
Hello Ryan,

Your suggestion worked. I first tried `pip install metapredict`, which gave me the Cython error. Then I updated cython followed by attempting to install metapredict again, returning the same error. After following your suggestions, everything was installed correctly.

I also tried updating metapredict (different system) with `pip install metapredict -U` and got the same cython error. However, I just uninstalled and reinstalled and that seemed to fix the problem.
